packages/prepare-commit-msg/src/utils/convert.ts

```diff
@@ -0,0 +1,61 @@
+import { type Author, coauthors } from "@coauthors/core";
+import { debug } from "./log";
+
+const COAUTHORS_PREFIX = "coauthors:";
+
+interface MessageInfo {
+  result: string;
+  isConverted: boolean;
+}
+
+export const convert = async (message: string): Promise<MessageInfo> => {
+  debug(`Original commit message: ${message}`);
+
+  const lines = message.split("\n");
+
+  const coauthorsLineIndex = lines.findIndex((line) =>
+    line.startsWith(COAUTHORS_PREFIX),
+  );
+  const coauthorsLine = lines.find((line) =>
+    line.trimStart().startsWith(COAUTHORS_PREFIX),
+  );
+
+  const authors = coauthorsLine
+    ?.substring(COAUTHORS_PREFIX.length, coauthorsLine.length)
+    .split(",")
+    .map(parseAuthorWithParentheses);
+
+  const coauthorsResult = authors ? await coauthors(authors) : undefined;
+
+  if (coauthorsLineIndex >= 0 && coauthorsResult) {
+    lines[coauthorsLineIndex] = coauthorsResult;
+  }
+
+  return {
+    isConverted: !!coauthorsLine,
+    result: lines.join("\n"),
+  };
+};
+
+const parseAuthorWithParentheses = (inputString: string): Author => {
+  const input = inputString.trim();
+  const parenthesesIndex = {
+    open: input.indexOf("("),
+    close: input.indexOf(")"),
+  };
+
+  if (parenthesesIndex.open >= 0 && parenthesesIndex.close >= 0) {
+    return {
+      user: input.slice(0, parenthesesIndex.open).trim(),
+      name: input
+        .slice(parenthesesIndex.open + 1, parenthesesIndex.close)
+        .trim(),
+    };
+  }
+
+  if (parenthesesIndex.open >= 0 && parenthesesIndex.close === -1) {
```

Comment:
If the user accidentally doesn't close it, do we really need to tell them this? We might be to consider using a more usable delimiter than parentheses.

Reply:
Could you suggest any alternatives by next Pull Request? 

Since we are still at version 0, it would be nice too to make suggestions with BREAKING CHANGES in the next PR after release new version. It is also true that it is still working well and there are no problems.

Reply:
I agree, it would be great to have a discussion about moving relevant functions to core